kokoro/config/test/ops_agent/release.gcl

```diff
@@ -0,0 +1,25 @@
+import 'common.gcl' as common
+import '../image_lists.gcl' as image_lists
+
+_parse_distro = lambda tag: {
+  parts = split(tag, '/')
+  ret = {
+    distro = parts[3]
+    arch = parts[4]
+  }
+}.ret
+
+config build = common.ops_agent_test {
+  params {
+    local _distro_arch = _parse_distro(params.environment._LOUHI_TAG_NAME)
+    platforms = image_lists.get(_distro_arch.distro + '_' + _distro_arch.arch).release
```

Comment:
This .get() needs a default argument. Like so:

```
    platforms = image_lists.get(
      _distro_arch.distro + '_' + _distro_arch.arch,
      'invalid_distro'
    ).release
```

Reply:
The default could be `null`, BTW…